src/mlpack/methods/logistic_regression/logistic_regression_function.hpp

```diff
@@ -28,10 +28,25 @@ template<typename MatType = arma::mat>
 class LogisticRegressionFunction
 {
  public:
+  /**
+   * Creates the LogisticRegressionFunction.
+   *
+   * @param predictors X, matrix of data points.
+   * @param responses y, the measured data for each point in X.
```

Comment:
I'd rather not give variable notations to the `predictors` and `responses`. Please make it something like
```
@param predictors The matrix of data points.
@param responses The measured data for each point in predictors.
```